recipes/tensorflow-addons/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "0.14.0" %}
+{% set tf_version_maj_min = "2.6" %}
+
+package:
+  name: tensorflow-addons
+  version: {{ version }}
+
+source:
+  url: https://github.com/tensorflow/addons/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: da9eaf683192054db73941e0de34d83754be4054dc92c409bc4693c3f3091586
+
+build:
+  number: 0
+  # missing dependency: https://github.com/conda-forge/tensorflow-feedstock/issues/124
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
+    - bazel
+    - bazel-toolchain
+    - rsync
+    - sed
+  host:
+    # GPU requirements for CUDA
+    - cudnn           # [cuda_compiler_version != "None"]
+    - python
+    - pip
+    - libprotobuf
+    # check tensorflow-range per addons-version, see
+    # https://github.com/tensorflow/addons#python-op-compatibility-matrix
+    - tensorflow {{ tf_version_maj_min }}.*
+    # needed for headers in tensorflow/core/framework
+    - libtensorflow_cc {{ tf_version_maj_min }}.*
+    - tensorflow =*=cpu*   # [cuda_compiler_version == "None"]
+    - tensorflow =*=cuda*  # [cuda_compiler_version != "None"]
+    - typeguard >=2.7
+  run:
+    - python
+    # due to lack of ABI, we need the same version at runtime, see
+    # https://github.com/tensorflow/addons#c-custom-op-compatibility-matrix
+    - tensorflow {{ tf_version_maj_min }}.*
+    - libtensorflow_cc {{ tf_version_maj_min }}.*
```

Comment:
TBH, I don't understand this - it's already there as a host req, so why does it also need to be in run? - but without this we get:
```
  ERROR (tensorflow-addons,lib/[something].so): Needed DSO lib/libtensorflow_framework.so.2 found in ['libtensorflow_cc']
  ERROR (tensorflow-addons,lib/[something].so): .. but ['libtensorflow_cc'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
```

Reply:
`libtensorflow_cc` doesn't have a `run_exports`, we should add one there.